.storybook/main.ts

```diff
@@ -54,21 +58,9 @@ const config: StorybookConfig = {
             },
             // Fix from: https://github.com/storybookjs/storybook/issues/25256#issuecomment-1866441206
             assetsInclude: ["/sb-preview/runtime.js"],
-            plugins:
-                configType === "PRODUCTION"
-                    ? [
-                          turbosnap({
-                              // This should be the base path of your storybook.  In monorepos, you may only need process.cwd().
-                              rootDir: config.root ?? process.cwd(),
-                          }),
-                      ]
-                    : [],
         });
     },
 
-    typescript: {
-        reactDocgen: "react-docgen",
```

Comment:
Storybook now defaults to `react-docgen` so no need to specify this here: https://storybook.js.org/docs/api/main-config/main-config-typescript#reactdocgen